ncat/ncat_main.c

```diff
@@ -618,6 +621,7 @@ int main(int argc, char *argv[])
 "      --proxy <addr[:port]>  Specify address of host to proxy through\n"
 "      --proxy-type <type>    Specify proxy type (\"http\" or \"socks4\" or \"socks5\")\n"
 "      --proxy-auth <auth>    Authenticate with HTTP or SOCKS proxy server\n"
+"  -z                         Just to establish connection, doesn't send any payload\n"
```

Comment:
Careful of the spacing here (6 spaces), and put this below `-w`. The help text for traditional and OpenBSD netcat calls this "zero-I/O mode [used for scanning]" so we should probably use something similar, though I would skip the mention of scanning. Perhaps: "Zero-I/O mode, report connection status only"